recipe/meta.yaml

```
@@ -1,34 +1,52 @@
-{% set version = "2.5.0" %}
+{% set version = "3.1.0" %}
 
 package:
   name: geojson
   version: {{ version }}
 
 source:
   url: https://pypi.io/packages/source/g/geojson/geojson-{{ version }}.tar.gz
```

For consistency with other recipes?
```suggestion
  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
```